File: R/tests/testthat/test_banditNarm2_kalman_filter.R
context("Test banditNarm2_kalman_filter")
library(hBayesDM)

test_that("Test banditNarm2_kalman_filter", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(banditNarm2_kalman_filter(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})

File: Python/tests/test_banditNarm2_kalman_filter.py
import pytest

from hbayesdm.models import banditNarm2_kalman_filter


def test_banditNarm2_kalman_filter():
    _ = banditNarm2_kalman_filter(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()
